baselines/pyproject.toml

```diff
@@ -37,13 +37,17 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8.3"
+python = "^3.7.12"
 # Mandatory dependencies
-flwr = "^0.17.0"
-# flwr = { path = "../", develop = true }  # For development
+#flwr = "^0.17.0"
+flwr = { path = "../", develop = true }  # For development
 torch = "^1.10.1"
 torchvision = "^0.11.2"
+omegaconf = { git = "https://github.com/pedropgusmao/omegaconf.git" } 
+hydra-core = { git = "https://github.com/pedropgusmao/hydra.git", branch="main" }
+ray = { extras = ["default"], version = "^1.9.1"}
```

Comment:
`pyproject.toml` in `main` branch now points to ray ^1.9.2

Reply:
Solved